graph_annotation.go

```diff
@@ -0,0 +1,18 @@
+package mackerel
+
+// GraphAnnotation parameters to post graph annotation
+type GraphAnnotation struct {
```

Comment:
I prefer fields are sorted according to the help document.

- title
- description
- from
- to
- service
- roles